content/blog/2020/04/2020-04-28-jenkins-is-the-way.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Call for User Stories - Jenkins is the Way"
+tags:
+- announcement
+author: alyssat
+opengraph:
+  image: /images/post-images/jenkins-is-the-way/jenkins-is-the-way-opengraph.png
+---
+
+image:/images/post-images/jenkins-is-the-way/jenkins-is-the-way.png[Jenkins Is The Way, role=center, float=right, height=284]
+
+One of the things we loved about going to developer conferences was meeting Jenkins users — newbies and old-timers alike — who are excited to talk about their projects and share tips on how to move forward using Jenkins.
+Since the coronavirus pandemic, we're learning to rely more on new ways to gather, and it's happening via Jenkins online meetups, GitHub collaborations, and Twitter threads, to name a few.
+
+It's a significant change.
+But what hasn't changed is the need to share stories about the things users have built, the solutions they've developed, and the excellent results they're getting from some really innovative Jenkins implementations.
+Then we wondered, why isn't anyone collecting these user stories and sharing them with the Jenkins community.
+
+== Introducing Jenkins is the Way
+
+So we took the first step to record and archive all the great stuff everyone in our community is building with Jenkins.
+This way, Jenkins users old and new can come to an archive and search for Jenkins solutions for inspiration.
+We foresee a vast library of solutions from all around the world, solving a wide array of challenges in every industry imaginable.
+We decided to call this archive "*Jenkins Is The Way*" and host it at https://JenkinsIsTheWay.io .
+
+To aggregate all these stories, we built a simple online questionnaire so that Jenkins users can submit their own experience using this leading open source automation server.
+With so many plugins to support building, deploying, and automating your projects, we expect to see a vast collection of stories.
+
+We've already received a handful, including stories that illustrate how *Jenkins Is The Way*:
+
+* to link:https://jenkinsistheway.io/story/code-your-own-release-pipelines[code your own release pipelines]
```

Comment:
These need to be replaced with the actual links.  Please do not merge until the links are replaced with the "real thing".

Reply:
The content looks great Mark. Will get you the links as soon as I can.